src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -568,7 +568,7 @@ protected function incrementOrDecrement($column, $amount, $extra, $method)
      */
     protected function incrementOrDecrementAttributeValue($column, $amount, $extra, $method)
     {
-        $this->{$column} = $this->{$column} + ($method === 'increment' ? $amount : $amount * -1);
+        $this->{$column} += $method === 'increment' ? $amount : $amount * -1;
```

Comment:
This changed was already tried to be changed in the past, without success #25411

Reply:
Why?